### PR TITLE
SA-2191 Metrics support aggregate by Mailbox Provider

### PR DESCRIPTION
**Pull request by jtianSP**

### CODE REVIEW

File: content/api/metrics.apib [hunk 1]
```diff
@@ -1348,6 +1348,121 @@ in the world.
...
++ Response 200 (application/json)
+
+        {
+            "results": [
+                {
+                    "count_injected": 109849,
+                    "count_targeted": 110923,
+                    "count_rejected": 1074,
+                    "count_sent": 107610,
+                    "mb_provider": "Verizon Media Group"
```
Comment (beardyman) [inline content/api/metrics.apib:1432]: Should we be returning this as `mailbox_provider` to stay consistent with the endpoints and query parameters that we're using?

Comment (jtianSP) [reply content/api/metrics.apib]: Good Catch, updated!

File: content/api/metrics.apib [hunk 2]
```diff
@@ -1348,6 +1348,121 @@ in the world.
...
+
+        Precision of the time window (`from`, `to`) bounds. All values valid up to a 24 hour window, except for `hour` which is also valid up a 31 day window.
+        See: [Precision Parameter](#header-precision-parameter).
+
+        + Values
+            + `1min`
+            + `5min`
+            + `15min`
+            + `hour`
+
```
Comment (tathibo1) [inline content/api/metrics.apib:1376]: Should include "day" here. See https://github.com/SparkPost/developers.sparkpost.com/blob/6a550a11ade6ea34d336ff07545d83281c35471a/content/api/metrics.apib#L493 for an example.

File: content/api/metrics.apib [hunk 3]
```diff
@@ -1348,6 +1348,121 @@ in the world.
...
+    + domains (optional, list) ... Delimited list of recipient domains to include.
+    + campaigns (optional, list) ... Delimited list of campaigns to include.
+    + templates (optional, list) ... Delimited list of template IDs to include.
+    + sending_ips (optional, list) ... Delimited list of sending IPs to include.
+    + ip_pools (optional, list) ... Delimited list of IP pools to include.
+    + sending_domains (optional, list) ... Delimited list of sending domains to include.
+    + subaccounts (optional, list) ... Delimited list of subaccount IDs to include.
+    + precision = `1min` (optional, enum) ...
+
+        Precision of the time window (`from`, `to`) bounds. All values valid up to a 24 hour window, except for `hour` which is also valid up a 31 day window.
```
Comment (tathibo1) [inline content/api/metrics.apib:1368]: Should include explanation of "day" precision returning times in UTC. See https://github.com/SparkPost/developers.sparkpost.com/blob/6a550a11ade6ea34d336ff07545d83281c35471a/content/api/metrics.apib#L485

File: content/api/metrics.apib [hunk 4]
```diff
@@ -1348,6 +1348,121 @@ in the world.
...
+            + `count_unique_rendered`
+            + `count_unique_confirmed_opened`
+            + `count_clicked`
+            + `count_unique_clicked`
+            + `count_targeted`
+            + `count_sent`
+            + `count_accepted`
+            + `count_spam_complaint`
+
+    + timezone =`UTC` (optional, string) ... Standard timezone identification string.
```
Comment (tathibo1) [inline content/api/metrics.apib:1411]: Should include a note about timezone being forced to UTC for day. See https://github.com/SparkPost/developers.sparkpost.com/blob/6a550a11ade6ea34d336ff07545d83281c35471a/content/api/metrics.apib#L535

**Review by tathibo1 — APPROVED** [review]
LGTM

**Pull request merged**